docs/docs/index.md

```
@@ -6,24 +6,50 @@ sidebar_position: 1
+
+| Binding               | Status        | Docs                                                      |
+| :-------------------- | :------------ | :-------------------------------------------------------- |
+| **JavaScript (WASM)** | ✅ Production | [JS API Reference](/docs/js/js-api-reference)             |
+| **C++**               | ✅ Production | [C++ API Reference](/docs/cpp/cpp-api-reference)          |
+| **C**                 | ✅ Production | [C API Reference](/docs/c/c-api-reference)                |
+| **Python**            | 🟡 Early      | [Python API Reference](/docs/python/python-api-reference) |
 
-- [JavaScript](./js)
+## Changelog
 
 ## Changelog
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)